invisible_cities/io/mcinfo_io_test.py

```diff
@@ -214,3 +214,10 @@ def test_load_sensors_data(mc_sensors_nexus_data):
     samples = list(zip(bins, wvf.charges))
 
     assert np.allclose(samples, sipm)
+
+    with tb.open_file(efile, mode='r') as h5in:
```

Comment:
This seems to have been added to an existing test, which suggests that this test now (and perhaps even before) tests for more than one thing. Would it be too much trouble to split this big test into separate smaller tests?

Reply:
Done!